onebusaway-api-webapp/src/main/java/org/onebusaway/api/actions/api/gtfs_realtime/TripUpdatesForAgencyAction.java

```diff
@@ -28,19 +29,25 @@
 import com.google.transit.realtime.GtfsRealtime.VehicleDescriptor;
 import org.onebusaway.util.AgencyAndIdLibrary;
 
+import javax.servlet.http.HttpServletResponse;
 import java.text.SimpleDateFormat;
-import java.util.ArrayList;
-import java.util.Date;
-import java.util.List;
+import java.util.*;
 
 public class TripUpdatesForAgencyAction extends GtfsRealtimeActionSupport {
 
   private static final long serialVersionUID = 2L;
 
+  private static final SimpleDateFormat _sdf;
+
+  static {
+    _sdf = new SimpleDateFormat("EEE, dd MMM yyyy HH:mm:ss 'GMT' ", Locale.US);
+    _sdf.setTimeZone(TimeZone.getTimeZone("GMT"));
```

Comment:
Is there a reason you chose this date format over ISO8601?  And why did you choose GMT over local?

Reply:
Sorry, I missed that.  If John recommended a date format than please use that.